cypress/objects/timeLoggingPage.js

```diff
@@ -0,0 +1,39 @@
+class TimeLoggingPage {
+  visit() {
+    cy.visit('/time-logging');
+  }
+
+  getUrl() {
+    return cy.url();
+  }
+
+  getPageTitle() {
+    return cy.get('.page__title');
+  }
+
+  getCalendar() {
+    return cy.get('.calendar');
+  }
+
+  getTileForm() {
+    return cy.get('.tile.form');
+  }
+
+  getUserInfoTitle() {
+    return cy.get('.user-info__title');
+  }
+
+  getMainNav() {
```

Comment:
Same as above, one might ask: what is a 'main nav'? Instead we could name it `getNavigationBar()` or something similar, so it's clear what the function refers to :)

Reply:
Fixed :)